docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+are updating your channel as part of [Upgrading to the latest version of Fabric](./upgrade_to_newest_version.html)
+you will need to set the capabilities to the levels appropriate to that release):
+
+```
+   {
+     "channel": {
+         "mod_policy": "Admins",
+             "value": {
+                 "capabilities": {
+                     "V1_4_3": {}
+                 }
+             },
+         "version": "0"
+     },
+     "orderer": {
+         "mod_policy": "Admins",
+             "value": {
+                 "capabilities": {
+                     "V1_4_2": {}
+                 }
+             },
+         "version": "0"
+     },
+     "application": {
+         "mod_policy": "Admins",
+             "value": {
+                 "capabilities": {
+                     "V1_4_2": {}
+                 }
+             },
+         "version": "0"
+     }
+   }
+```
+
+Note that by default peer organizations are not admins of the orderer system
+channel and will therefore be unable to propose configuration updates to it. An
+orderer organization admin would have to create a file like this (without the
+`application` group capability, which does not exist in the system channel) to
+propose updating the system channel configuration. Note that because application
+channel copy the system channel configuration by default, unless a different
+channel profile is created which specifies capability levels, the `Channel` and
+`Orderer` group capabilities for the application channel will be the same as those
+in the network's system channel.
+
+## Orderer system channel capabilities
+
+Because application channels copy the configuration of the orderer system channel
+by default, it is considered a best practice to update the capabilities of the
+system channel before any application channels. This mirrors the process of
+updating ordering nodes to the newest version before peers, as described in
+[Upgrading your components](./upgrading_your_components.html).
+
+Note that the orderer system channel is administered by ordering service
+organizations. By default this will be a single organization (the organization
+that created the initial nodes in the ordering service), but more organizations
+can be added here (for example, if multiple organizations have contributed nodes
+to the ordering service).
+
+**Make sure all of the ordering nodes in your ordering service have been upgraded
+to the required binary level before updating capabilities. If an ordering node
+is not at the required level, it will be unable to process the config block with
+the capability and will crash**. Similarly, note that if a new channel is created
+on this ordering service, all of the peers that will be joined to it must be at
+least to the node level corresponding to the `Channel` capability, otherwise
+they will also crash. For more information, check out [Capabilities](./capabilities_concept.html).
```

Comment:
Orderers need to at least be at the Orderer and Channel capability version.
Peers need to at least be at the Application and Channel capability version.